unityfiles/Assets/EnemyHealthBarGUIController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Takes care of displaying and updating the enemy's health bar
+/// </summary>
+public class EnemyHealthBarGUIController : MonoBehaviour {
+    
+    [SerializeField]
+    private GameObject healthBarBackground;
+
+    private float startHealth;
+
+    [SerializeField]
+    private Image healthBar;
```

Comment:
er det ikke enklere og finne disse to med Go.trygetchildrenlike

Reply:
Bedre å sette manuelt en å prøve å finne, siden det blir en veldig skjult dependency, som man ikke ser før man starter, og da ikke har dem.

Reply:
Nå er healthBar componenten en ganske self contained greie. Så tenkte det var greiere å hooke det opp i Unity. Da kan man bare droppe HealthBar.prefab inn i en ny enemy.